### PR TITLE
Created Javascript file for Popup window

### PR DESCRIPTION
**Pull request by GintsM**
All requirements are deployed. 
A Popup window appears on top of each card. All popup windows show exact content from it appears. 
Very easy is to add change content for card or create a new card I use a constructor-based javascript code.

### CODE REVIEW

File: card.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
+function Card(title, expierence, image) {
```
Comment (carlos-ssh) [inline card.js:1]: **Kindly work with the requested feature in your project requirements:**
- In order to associate each project with the popup window details, you must use a JavaScript array to store all of the information for all projects.

The JavaScript code implements an array of objects to store JavaScript objects containing the information for each project.

Check this example:
```
const projects = [
 {
   name: "Submarine game",
    description: "A Phaser3 plataformer game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
 }
 {
   name: "Submarine game",
    description: "A Phaser3 plataformer game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
 }
]
```

**Reminder:**
- Modal popup data is populated with the data from the projects array.

File: .gitignore [hunk 2]
```diff
@@ -3,4 +3,6 @@ node_module
 .hintrc
 LICENSE
 style.css
-mobile.css
\ No newline at end of file
+mobile.css
+launch.json
+settings.json
```
Comment (carlos-ssh) [inline .gitignore:8]: [OPTIONAL]
**Keep your repo clean and professional**
- Add external directories and files (``.vscode``) to your ``.gitignore`` file.

File: .gitignore [hunk 3]
```diff
@@ -3,4 +3,6 @@ node_module
 .hintrc
 LICENSE
```
Comment (carlos-ssh) [inline .gitignore:4]: - Provide a link to your ``MIT`` license in your ``README`` file.

File: .gitignore (hunk 2 above)
Comment (GintsM) [reply .gitignore:8]: Those files are for my working place and not related to repo at all. That's why I use gitignore to not pollute repo with unnecessary files. Or do you have other suggestions?

**Pull request merged**